components/executors/k8s/private-docker-registry/private-docker-registry.PersistentVolumeClaim.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: private-docker-registry
+  labels:
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
+    app.kubernetes.io/component: private-docker-registry
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      # To be adjusted based on the number and size of images used in batch changes and auto-indexing
+      storage: 100Gi
+  storageClassName: sourcegraph
```

Comment:
Not sure if this matters, but I removed all the storageClassName from the base [(pgsql for example](https://github.com/sourcegraph/deploy-sourcegraph-k8s/blob/main/base/sourcegraph/pgsql/pgsql.PersistentVolumeClaim.yaml)) so that users can update it using one of our [storage-class components](https://github.com/sourcegraph/deploy-sourcegraph-k8s/blob/main/components/storage-class/sourcegraph/kustomization.yaml)